shell/platform/android/android_surface_gl_skia.cc

```diff
@@ -167,6 +166,8 @@ GLFBOInfo AndroidSurfaceGLSkia::GLContextFBO(GLFrameInfo frame_info) const {
   // The default window bound framebuffer on Android.
   return GLFBOInfo{
       .fbo_id = 0,
+      .partial_repaint_enabled = onscreen_surface_->SupportsPartialRepaint(),
```

Comment:
The version in place before https://github.com/flutter/engine/pull/35022 landed was this https://github.com/flutter/engine/pull/35022/files#:~:text=framebuffer%20on%20Android.-,return%200%3B,-return%20GLFBOInfo%7B. It did not pass any sort of information besides the fbo_id, which makes me think that this might not be the problem yet.